From e25ddb1dae63d403cd7d77d1d4419089fd3b81b8 Mon Sep 17 00:00:00 2001
From: Sam Ravnborg <srn@skov.dk>
Date: Fri, 17 Feb 2017 21:37:43 +0100
Subject: [PATCH] arm9-cpu: clanup config file

Drop unused stuf from include file.
No need to carry around obsolete NAND + DataFlash + RTC stuff

Disable debug output (do not define DEBUG)
---
 include/configs/arm9-cpu.h | 19 +------------------
 1 file changed, 1 insertion(+), 18 deletions(-)

diff --git a/include/configs/arm9-cpu.h b/include/configs/arm9-cpu.h
index 2dd5e64..e30e890 100644
--- a/include/configs/arm9-cpu.h
+++ b/include/configs/arm9-cpu.h
@@ -5,8 +5,6 @@
 #ifndef __CONFIG_H
 #define __CONFIG_H
 
-#define DEBUG 1
-
 #define CONFIG_AT91_LEGACY
 /*
  * SoC must be defined first, before hardware.h is included.
@@ -115,7 +113,6 @@
 
 #define CONFIG_CMD_PING		1
 #define CONFIG_CMD_DHCP		1
-//#define CONFIG_CMD_NAND		1
 #define CONFIG_CMD_USB		1
 
 /** 
@@ -134,8 +131,8 @@
 #define I2C_SCL(bit)            at91sam9263_i2c_scl(bit)
 #define I2C_SDA(bit)            at91sam9263_i2c_sda(bit)
 #define I2C_READ                at91sam9263_i2c_read()
+
 /* Watchdog */
-#define CONFIG_RTC_M41T82_WATCHDOG      1
 #define CONFIG_RTC_DS1374_WATCHDOG      1
 #define CONFIG_SYS_I2C_RTC_ADDR         0x68
 #define CONFIG_HW_WATCHDOG              1
@@ -150,16 +147,6 @@
 #define CONFIG_SYS_INIT_SP_ADDR \
 	(ATMEL_BASE_SRAM1 + 0x1000 - GENERATED_GBL_DATA_SIZE)
 
-/* DataFlash */
-// #define CONFIG_ATMEL_DATAFLASH_SPI
-// #define CONFIG_HAS_DATAFLASH		1
-// #define CONFIG_SYS_SPI_WRITE_TOUT		(5*CONFIG_SYS_HZ)
-// #define CONFIG_SYS_MAX_DATAFLASH_BANKS		1
-// #define CONFIG_SYS_DATAFLASH_LOGIC_ADDR_CS0	0xC0000000	/* CS0 */
-// #define AT91_SPI_CLK			15000000
-// #define DATAFLASH_TCSS			(0x1a << 16)
-// #define DATAFLASH_TCHS			(0x1 << 24)
-
 /* NOR flash, if populated */
 #define CONFIG_SYS_FLASH_CFI			1
 #define CONFIG_FLASH_CFI_DRIVER			1
@@ -173,11 +160,7 @@
 #define CONFIG_SYS_MONITOR_BASE	CONFIG_SYS_FLASH_BASE
 #define CONFIG_SYS_MONITOR_LEN	(256 << 10)
 
-// #define CONFIG_ENV_IS_IN_DATAFLASH
-
 #define CONFIG_ENV_IS_IN_FLASH
-// #define CONFIG_ENV_SIZE                         (16 * 1024)
-// #define CONFIG_ENV_ADDR                         (CONFIG_SYS_DATAFLASH_LOGIC_ADDR_CS0 + 0x0007FE00)
 /* Address and size of Primary Environment Sector */
 #define CONFIG_ENV_SIZE			0x000004000 /**< Must be smaller that 64KB otherwise h_export fails*/
 #define CONFIG_ENV_ADDR                   (CONFIG_SYS_FLASH_BASE + 0x00080000)
-- 
1.8.3.1

